packages/xo-web/src/common/intl/messages.js

```diff
@@ -769,8 +770,10 @@ const messages = {
   cloneVmLabel: 'Clone',
   cleanVm: 'Clean VM directory',
   fastCloneVmLabel: 'Fast clone',
+  startMigratedVm: 'Start the migrated VM',
   vmConsoleLabel: 'Console',
   vmExportUrlValidity: 'The URL is valid once for a short period of time.',
+  vmLukewarmMigration: 'Lukewarm migration',
```

Comment:
"Warm" might be enough. It's still colder than live/hot migration and hotter than cold migration.